src/block-data-providers/bitcoin-core/provider.ts

```diff
@@ -243,4 +273,8 @@ export class BitcoinCoreProvider
     private convertToSatoshi(amount: number): number {
         return currency(amount, { precision: 8 }).multiply(SATS_PER_BTC).value;
     }
+
+    private hasFullVerboseInput(version: number): boolean {
+        return version >= BITCOIN_CORE_FULL_VERBOSITY_VERSION;
+    }
```

Comment:
Let's change it to this.

```suggestion
    private versionToVerbosity(version: number): 2 | 3 {
        return version >= BITCOIN_CORE_FULL_VERBOSITY_VERSION ? 3 : 2;
    }
```